src/js/components/FilterByTaskState.js

```diff
@@ -56,13 +57,22 @@ class FilterByTaskState extends React.Component {
   }
 
   render() {
+    let {inverseStyle} = this.props;
+
+    let dropdownMenuClassSet = classNames({
```

Comment:
yes, and also at times we need to pass the inverse styles down through many child components (`NodeDetailPage` is inverse and passes the inverse flag to its child `TaskView` which passes to its child `TaskTable`), so I think it's meaningful to have this specific inverse flag